x/slashing/keeper/infractions.go

```diff
@@ -2,11 +2,10 @@ package keeper
 
 import (
 	"fmt"
-	"github.com/tendermint/tendermint/crypto"
-	db "github.com/tendermint/tm-db"
-
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/slashing/types"
+	"github.com/tendermint/tendermint/crypto"
+	db "github.com/tendermint/tm-db"
 )
 
 func (k Keeper) HandleValidatorSignature(ctx sdk.Context, batch db.Batch, addr crypto.Address, power int64, signed bool, blockCount int64, slashable bool) {
```

Comment:
Line 25: Github does not allow inserting a comment on unmodified code, there's a Todo for emitting an EventTypeLiveness that can be copied from the Sdk.
Line 35: another Todo

Reply:
Added Liveness event and removed TODOs.